web/src/routes/OrganizationsList/Organization/Tabs/Settings/Settings.tsx

```diff
@@ -7,47 +7,158 @@ import {
   FlexItem,
   FormGroup,
   Form,
+  FormAlert,
   TextInput,
   FormSelect,
   FormSelectOption,
   ActionGroup,
   Button,
+  Alert,
+  AlertGroup,
 } from '@patternfly/react-core';
 import {useLocation} from 'react-router-dom';
 import {useCurrentUser} from 'src/hooks/UseCurrentUser';
 import {useOrganization} from 'src/hooks/UseOrganization';
+import {useOrganizationSettings} from 'src/hooks/UseOrganizationSettings';
+import {IOrganization} from 'src/resources/OrganizationResource';
+import {humanizeTimeForExpiry, getSeconds, isValidEmail} from 'src/libs/utils';
+import {addDisplayError} from 'src/resources/ErrorHandling';
+
+type validate = 'success' | 'warning' | 'error' | 'default';
+const timeMachineOptions = {
```

Comment:
This is actually configurable from the backend via `TAG_EXPIRATION_OPTIONS`. You should be able to read in that value from the quay config